test/DefaultPrompt.Tests.ps1

```diff
@@ -531,17 +533,12 @@ M test/Baz.Tests.ps1
         Context 'Removing the posh-git module' {
             It 'Correctly reverts the Window Title back to original state' {
                 Set-Item function:\prompt -Value ([Runspace]::DefaultRunspace.InitialSessionState.Commands['prompt']).Definition
+                $originalTitle = & $module Get-Variable OriginalWindowTitle -ValueOnly
```

Comment:
Why not?
```
$originalTitle = & $module { $OriginalWindowTitle }
```

Reply:
But do it after importing posh-git

Reply:
Ah, now I see you're capturing this variable in the test fixture before the test runs.

Reply:
> Why not?
> 
> ```
> $originalTitle = & $module { $OriginalWindowTitle }
> ```

I don't use this syntax often, so I refer to https://github.com/dahlbyk/posh-git/blob/5a87e8a196d13395f978d774f80dedb40d43b85e/src/Utils.ps1#L7